bukkit/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
     implementation ('com.zaxxer:HikariCP:5.0.1') {
         exclude module: 'slf4j-api'
     }
+    implementation 'com.github.Emibergo02:RedisEconomy:2.1-SNAPSHOT'
```

Comment:
Why are we implementing it? This requires the user to have the plugin installed, yeah? So we don't need to bundle it with this jar. Change to compileOnly and remove the relocate.